services/common/src/components/permits/PermitConditionCategory.tsx

```diff
@@ -56,7 +56,7 @@ export const EditPermitConditionCategoryInline: FC<IPermitConditionCategoryProps
     setIsEditMode(false);
   }
 
-  const titleElement = <Typography.Title style={{ marginBottom: 0 }} level={3}>{formatPermitConditionStep(category.step)} {category.description} ({props.conditionCount})</Typography.Title>;
+  const titleElement = <span style={{ marginBottom: 0 }}>{formatPermitConditionStep(category.step)} {category.description} ({props.conditionCount})</span>;
```

Comment:
I was getting console errors that h3 cannot be a descendent of h3, so I made it a span. Looks the same. It's visible what's happening in the snapshot changes.